python/katana/local/analytics/_wrappers.pyx

```diff
@@ -37,17 +37,17 @@ cdef unique_ptr[NUMAArray[uint64_t]] handle_result_unique_numaarray_uint64(Resul
     return move(res.value())
 
 
-# "Algorithms" from PropertyGraph
+# "Algorithms" from Graph
 
-cdef extern from "katana/PropertyGraph.h" namespace "katana" nogil:
+cdef extern from "katana/Graph.h" namespace "katana" nogil:
```

Comment:
double checking if .h was renamed

Reply:
It was not. Ooops. How did this even build?!?! :-/ I'll fix this.